src/agent/sqlite_manager/include/sqlite_manager.hpp

```diff
@@ -1,10 +1,34 @@
 #pragma once
+
+#include <SQLiteCpp/SQLiteCpp.h>
+#include <SQLiteCpp/VariadicBind.h>
+
+#include <string>
+
 namespace sqlite_manager
 {
+    class Column
+    {
+        std::string strName;
```

Comment:
We shouldn't use Hungarian notation if possible.

Reply:
Old habits die hard... Corrected.